hiera/hosts/vagrant-test.tag1consulting.dev.yaml

```diff
@@ -0,0 +1,106 @@
+# Configuration for vagrant-test.tag1consulting.dev
+#
+# This file contains example Hiera configuration that can be used to
+# override or append to default settings in common.yaml.
+# In order to be pulled in by Hiera, the file must be named the same as your VM FQDN
+# (configured in Vagrantfile.local, or defaults to vagrant-test.tag1consulting.dev)
+# with a .yaml extension. e.g. vagrant-test.tag1consulting.dev.yaml
+
+#################################################
+# Apache configuration.
+
+# Example 1: expand the default vhost definition to include an additional
+# <Location> block and an alias to access it.
+#
+# The hostname 'vagrant-test.tag1consulting.dev' matches the definition in common.yaml.
+site_profile::web::vhosts:
+  vagrant-test.tag1consulting.dev:
+    port: 80
+    docroot: /var/www/html
+    docroot_owner: 'apache'
+    docroot_group: 'vagrant'
+    docroot_mode: '2775'
+    directories:
+      - path: '/var/www/html'
+        allow_override: All
+        options:
+          - Indexes
+          - FollowSymLinks
+          - MultiViews
+
+# Example 3: Create your own vhost and set is as the default apache vhost.
+# This will allow your vhost to match any hostname except for 'vagrant-test.tag1consulting.dev'.
+#
+#site_profile::web::vhosts:
+#  vagrant-test.tag1consulting.dev:
+#    default_vhost: false
+#
+#  new-default.tag1consulting.dev:
+#    port: 80
+#    default_vhost: true
+#    docroot: /var/www/example3
+#    directories:
+#      - path: '/var/www/example3'
+#        allow_override: All
+#        options:
+#          - Indexes
+#          - FollowSymLinks
+#          - MultiViews
+
+#################################################
+# MySQL configuration.
+
+mysql::server::databases:
+  'vagrant':
+    ensure: 'present'
+    charset: 'utf8'
+
+mysql::server::users:
+  'vagrant@localhost':
+    ensure: 'present'
+    password_hash: '*04E6E1273D1783DF7D57DC5479FE01CFFDFD0058'
+
+mysql::server::grants:
+  'vagrant@localhost/vagrant.*':
+    ensure: 'present'
+    user: 'vagrant@localhost'
+    privileges: ['ALL']
+    table: 'vagrant.*'
+
+
+#################################################
```

Comment:
Which part is unused?  The mysql::server::grants is needed to allow the vagrant user access to the vagrant DB.